integration/src/wallets/xdefi.ts

```diff
@@ -1,19 +1,106 @@
+};
 
 export function name(): string {
   return "XDeFi";
 }
 
 export async function createWallet(): Promise<core.HDWallet> {
-  const wallet = createMockWallet();
-  if (!wallet) throw new Error("No XDeFi wallet found");
+  // mock xdefi
+  (globalThis as any).xfi = {
```

Comment:
`packages/hdwallet-xdefi/src/adapter.ts` has the only occurrence of `globalthis as any` in the codebase currently - don't we have any way to avoid these two `any`s?